kapitan/resources.py

```diff
@@ -55,9 +55,11 @@ def resource_callbacks(search_paths):
                           partial(inventory, search_paths)),
             "file_read": (("name",),
                           partial(read_file, search_paths)),
-            "yaml_dump": (("obj",), yaml_dump),
             "sha256_string": (("obj",), sha256_string),
             "gzip_b64": (("obj",), gzip_b64),
+            "yaml_dump": (("obj",), yaml_dump),
+            "yaml_to_json": (("name",),
```

Comment:
Rename to `yaml_load`?

Reply:
`yaml_to_json` tells us exactly  functionality of this function. Though yaml_load matches other functions in naming style.

Reply:
it's explicitly used for jsonnet anyway so this yaml_load makes sense and matches the style of other functions. :)

Reply:
Let's keep the naming style :)